app/src/lib/rules/rules_parser.js

```diff
@@ -349,7 +333,7 @@ export function create_token_transform(transform_json) {
  * @returns {LookupFilter}
  */
 function has_usage(char) {
-	return result => result.categorization.includes(char)
+	return ({ concept }) => concept !== null && concept.categorization.includes(char)
```

Comment:
```suggestion
	return ({ concept }) => concept?.categorization.includes(char)
```

Reply:
The return type of `LookupFilter` expects a boolean. Using the `?` makes it nullable, so I'd have to add `?? false` at the end, which I do not like better than the explicit null check.